src/app/zap-templates/zcl/data-model/silabs/zll.xml

```diff
@@ -185,13 +185,17 @@ limitations under the License.
       <arg name="enhancedHue" type="INT16U"/>
       <arg name="direction" type="HueDirection"/>
       <arg name="transitionTime" type="INT16U"/>
+      <arg name="optionsMask" type="BITMAP8" introducedIn="matter-spec" optional="1"/>
+      <arg name="optionsOverride" type="BITMAP8" introducedIn="matter-spec" optional="1"/>
```

Comment:
Those args are in ZCL8 as far as I can tell.  That applies throughout this file.  So not sure we should claim `introducedIn="matter-spec"`.

Reply:
I changed it to zcl8. I looked but wasn't sure of the exact zcl8 revision file when it was added.